tests/server-function/src/app.tsx

```diff
@@ -1,19 +1,34 @@
+import { join } from 'path';
```

Comment:
I wonder if it would make sense to also include a similar test, but with an npm library instead of a built-in 🤔. Bundlers often have different logic/assumptions when it come to built-ins vs npm libraries. By testing both import types we could probably catch more potential issues.

Reply:
I'll add a lodash example